src/Tribe/Editor.php

```diff
@@ -402,10 +450,21 @@ public function is_edit_screen() {
 		return 'post' === $current_screen->base;
 	}
 
+	/**
+	 * Check whether the current page is the Full Site editor.
+	 *
+	 * @since 5.13.0
+	 *
+	 * @return bool
+	 */
+	public function is_full_site_editor() {
+		return Tribe__Admin__Helpers::instance()->is_screen( 'site-editor' );
```

Comment:
The `Tribe__Admin__Helpers:is_screen` method does a bit of computation behind the scenes: cache the value on first fetch here.